mage_ai/io/base.py

```diff
@@ -30,11 +31,34 @@ class FileFormat(str, Enum):
 }
 
 
+class IOPrintHandler:
```

Comment:
Is this specific to the io module? Can it be generalized and put into the logger file? https://github.com/mage-ai/mage-ai/blob/master/mage_ai/shared/logger.py
Let's centralize the logging related classes and methods.

Reply:
Moved into `logger`.